### ISSUE TITLE
Escort NPCs aggro'ing to any mob that is in combat

### ISSUE THREAD

**Issue by gotmilk0112** [LightsHope/server#1035]
This has been bugged for years across multiple private servers.

While on an escort quest, if ANYONE, even people outside of your group, aggros a mob within ~50yd of the escort NPC, the escort will aggro to that mob and try to fight it until it's dead.

This often leads to the NPC dying because it unwittingly pulls multiple mobs, or gets the attention of an enemy player which ends up killing it.

**Comment by ifyousayso** [comment]
Yes, this was broken about a week ago with the turtle escort in Tanaris.

**Comment by gotmilk0112** [comment]
It does it on literally every escort in the game.
